frontend/schemaCMS/src/routes/page/editPage/editPage.component.js

```diff
@@ -254,6 +275,36 @@ export const EditPage = ({
           </NextButton>
         </ModalActions>
       </Modal>
+      <Modal
+        ariaHideApp={false}
+        isOpen={leavingPageModalOpen}
+        contentLabel={messages.confirmLeavingPage}
+        style={modalStyles}
+      >
+        <ModalTitle>
+          <FormattedMessage {...messages.leavingPageTitle} />
+        </ModalTitle>
+        <ModalActions>
+          <BackButton
+            onClick={() => {
+              setLeavingPageModalOpen(false);
+              setCustomLocation(false);
+            }}
```

Comment:
Here too, we need to move this to a named function

Reply:
done